otherlibs/stdune/src/monoid.mli

```diff
@@ -6,7 +7,6 @@ module type S = Monoid_intf.S
     operator synonym [( @ ) = combine], as well as derived functions [reduce]
     and [map_reduce]. *)
 module Make (M : Basic) : S with type t := M.t
-[@@inlined always]
```

Comment:
I think this isn't doing anything. Please correct me if I'm wrong.

Reply:
Perhaps it does something in your internal fork of OCaml? I'd imagine flambda pays more attention to these annotations.

Reply:
Actually, I removed it because internally I'm getting an error:

> the "inlined" attribute cannot appear in this context

which presumably means we enable some of the warnings that aren't enabled by default in the Dune repo. Maybe we should enable them to clean up such needless annotations.